src/tasks/RuntimeConfigParser/RuntimeConfigParser.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Reflection.Metadata;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+public class RuntimeConfigParserTask : Task
+{
+    /// <summary>
+    /// The path to runtimeconfig.json file.
+    /// </summary>
+    [Required]
+    public string RuntimeConfigFile { get; set; } = "";
+
+    /// <summary>
+    /// The path to the output binary file.
+    /// </summary>
+    [Required]
+    public string OutputFile { get; set; } = "";
+
+    /// <summary>
+    /// List of reserved properties.
```

Comment:
Per Jan's comment, it's probably worth being clearer about what "reserved" means here.

Reply:
Updated. Thanks!